src/methods/is.js

```diff
@@ -109,7 +109,8 @@ export function IsCallable(realm: Realm, _func: Value): boolean {
   if (func.isSimpleObject()) return false;
 
   // 2. If argument has a [[Call]] internal method, return true.
-  func = func.throwIfNotConcreteObject();
+  if (func instanceof AbstractObjectValue) func = func.getTemplate();
```

Comment:
This `getTemplate()` function is confusing. It only returns if the template is partial but not if all fields are known?

Reply:
Makes sense. Changed code.